Gemfile.lock

```diff
@@ -0,0 +1,32 @@
+PATH
+  remote: .
+  specs:
+    slop (4.3.0)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    byebug (11.1.3)
+    coderay (1.1.3)
+    method_source (1.1.0)
+    minitest (5.0.8)
+    pry (0.14.2)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    pry-byebug (3.10.1)
+      byebug (~> 11.0)
+      pry (>= 0.13, < 0.15)
+    rake (13.2.1)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  minitest (~> 5.0.0)
+  pry (~> 0.14.2)
+  pry-byebug (~> 3.10.1)
+  rake
+  slop!
+
+BUNDLED WITH
+   1.17.2
```

Comment:
will this version have impact on other people's local set up? i think i was getting warnings before to upgrade to 2.5.6. so that's what i have in my Gemfile.lock but don't know if that matters

Reply:
we can, i was just using asdf originally, so i imagine this was pinned to whatever version of bundler was associated with the original ruby version for this repo. I'll update that and repush